### PR TITLE
Update page in Connect API Manager “agent-variables”

### PR DESCRIPTION
**Pull request by jcollins-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by jcollins-axway** [comment]
JIRA APIGOV-15742 and APIGOV-15735

**Comment by jcollins-axway** [comment]
@kflan-io @sbolosan @dfeldick - please review for content

File: content/en/docs/central/connect-api-manager/agent-variables.md [hunk 1]
```diff
@@ -87,8 +87,8 @@ You can extend the previous minimum variable list with the following variables.
...
+| CENTRAL_ORGANIZATIONID                        | The Organization ID from AMPLIFY Central. Locate this at Platform > User > Organization.                                                                                                                                                                                                                                                                                                                                                                                                                                                |   |
+| CENTRAL_TEAM                          | The name of the team in AMPLIFY Central that all APIs will be linked. Locate this at AMPLIFY Central > Access > Team Assets.).                                                                                                                                                                                                                                                                             |   |
```
Comment (sbolosan) [inline content/en/docs/central/connect-api-manager/agent-variables.md:91]: maybe... "The name of the team in AMPLIFY Central that all APIs will be linked to."

Comment (jcollins-axway) [reply content/en/docs/central/connect-api-manager/agent-variables.md]: Updated

**Pull request merged**